config/locales/candidate_interface/section_complete.yml

```diff
@@ -0,0 +1,8 @@
+en:
+  activemodel:
+    errors:
+      models:
+        candidate_interface/section_complete_form:
+          attributes:
+            completed:
+              blank: Select whether you have completed this section
```

Comment:
I think we also need a version of this validation error for the "reviewed" state?

Reply:
@frankieroberto discussed this and agreed it is not necessary as a candidate will have previously 'completed' a given section  (i.e. indicate whether they have completed it or not). We also have logic to indicate whether a section review is required (this sets the value of `completed` to `false`) - https://github.com/DFE-Digital/apply-for-teacher-training/pull/4551/files#diff-22fdd3b57e65e63df6dd2ff571a3632cbb3a7a45f1ec11fbb3b338753fdb6c4bR361